test.js

```diff
@@ -0,0 +1,36 @@
+const test = require('tape')
+const noNodeConfig = require('./index')
+const co = require('co')
+const config = require('./testConfig/configData')
+const keys = ['mongo', 'redis']
+const PROCESS_ENV = process.env.NODE_ENV = 'test_env'
+
+const configObject = co(function* () {
+    return yield noNodeConfig({config})
+})
+
+test('Should initialize config object', (t) => { 
+    t.plan(3)
```

Comment:
@anuragCES in both test cases I suggest to use `t.end()` instead. For me it's nicer + if it's used right it will help to prevent tests from hanging if `configObject` will fail (I'll explain in the next comment)